### DISCUSSION TITLE
Need help in bar chart grouped graph.

### DISCUSSION THREAD

**Discussion by zee-algo** [plouc/nivo#2344] [General]
I need to make following chart.

![image](https://github.com/plouc/nivo/assets/111589038/ec25b4bc-aa10-45cb-b7c5-af4bfeec0486)

Is this possible? If so, how can I achieve this?

**Comment by hyjoong** [comment]
You can check out the official documentation at https://nivo.rocks/bar for demos and testing purposes.

**Comment by ollwenjones** [comment]
couple things to consider
1. Nivo bars put the bar labels in the bars, not after them, so this is a difference you will have to work around with custom elements or let go of
2. you just need to group your data into series to get the grouped behavior your are after here, but...
3. ~~The axis ticks don't behave quite like that, so that might be something you compromise on, or you could just compound a bunch of charts together and use Axis legend for the labels flat along the bottom. Turn axis off except for the rightmost chart~~
strike through my last comment. You should be able to achieve ticks like that with similar props
```js
   axisBottom={{
        orient: "bottom",
        tickSize: 0,
        tickPadding: 10,
        tickRotation: 0,
```
